README.md

```diff
@@ -69,6 +69,8 @@ API | Description | Auth | HTTPS | CORS |
 | [HTTPCat](https://http.cat/) | Cat for every HTTP Status | No | Yes | Unknown |
 | [IUCN](http://apiv3.iucnredlist.org/api/v3/docs) | IUCN Red List of Threatened Species | `apiKey` | No | Unknown |
 | [Movebank](https://github.com/movebank/movebank-api-doc) | Movement and Migration data of animals | No | Yes | Unknown |
+| [PlaceDog](https://place.dog) | Placeholder Dog pictures | No | Yes | Yes |
+| [PlaceGOAT](https://placegoat.com/) | Placeholder goat images | No | Yes | Unknown |
```

Comment:
`https://placegoat.com/` doesn't seem to work. Can you please update the URL or remove the PlaceGOAT record?

Reply:
Yeah sure thing!